python/ray/data/_internal/planner/exchange/sort_task_spec.py

```diff
@@ -116,10 +117,11 @@ def __init__(
         self,
         boundaries: List[T],
         sort_key: SortKey,
+        batch_format: Optional[str],
```

Comment:
nit: i think we are guaranteed this is a string here. also consistent with `SortAggregateTaskSpec `
```suggestion
        batch_format: str,
```